src/index.html

```diff
@@ -5,8 +5,404 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="images/logo.png"
+        alt="#">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">
+              apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#" class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              video
+            </a>
+          </li>
+
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div class="catalog__card card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 inch; Retina 5K Monoblock (MNED2UA/A)"
+          class="card__img"
+        >
+
+        <p class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <div class="card__code">
+          Product code: 195434
+        </div>
+
+        <div class="card__mark">
+          <div class="card__stars stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__review">
+            Reviews: 5
+          </p>
+        </div>
+
+        <div class="card__price">
+          <div class="card__textprice">Price:</div>
+          <div class="card__valueprice">$2,199</div>
+        </div>
+
+        <a
+        class="card__button"
+        data-qa="card-hover"
+        href="#">
+          buy
```

Comment:
use 2 spaces (or 1 tab (if your tab in IDE settings is 2 spaces)) (do it in all of links in cards)